src/Test/StateMachine/Internal/Parallel.hs

```diff
@@ -254,6 +256,24 @@ toForkOfOps h = Fork (mkOps l) p' (mkOps r)
     dynResp = fromMaybe (error "toForkOfOps: impossible.") . fromDynamic
   mkOps _  = error "mkOps: Impossible."
 
+toBoxDrawings :: forall cmd. (IxFunctor cmd, ShowCmd cmd, HasResponse cmd) => History cmd -> Doc
```

Comment:
I put some stuff in https://github.com/advancedtelematic/quickcheck-state-machine/blob/master/src/Test/StateMachine/Internal/Types/IntRef.hs to avoid cycling imports, perhaps you can do a similar thing?